src/components/Settings/Export/Export.component.js

```diff
@@ -282,6 +296,44 @@ class Export extends React.Component {
                   </div>
                 </ListItemSecondaryAction>
               </ListItem>
+              <Divider />
```

Comment:
Im thinking we can improve it. Take a look at this: 
![image](https://github.com/cboard-org/cboard/assets/80465873/16dbb876-c36b-4403-8150-910982daa5e5)

What do you think? You can investigate on the `Settings.component.js`